eureka-webapp/src/main/webapp/eureka/config/app-properties.js

```diff
@@ -1,14 +1,18 @@
 (function() {
 
     'use strict';
+    
+    var eurekaModule = angular.module('eureka');
+
+    eurekaModule.service('AppPropertiesService',['$http', function($http){
+            return {
+                    getAppProperties:function() {
+                                    return $http.get('/eureka-services/api/appproperties');
+                            }
```

Comment:
Also, eureka-services is hard-coded there. This should be going through the proxy.